jekyll/_cci2/server/installation/phase-1-prerequisites.adoc

```diff
@@ -0,0 +1,821 @@
+---
+version:
+- Server v4.x
+- Server Admin
+---
+= Phase 1 - Prerequisites
+:page-layout: classic-docs
+:page-liquid:
+:page-description: Find the general and infrastructure-specific requirements that are needed in order to configure the CircleCI server application.
+:icons: font
+:toc: macro
+:toc-title:
+:relfileprefix: ../
+:relfilesuffix: /
+
+// This doc uses ifdef and ifndef directives to display or hide content specific to Google Cloud Storage (env-gcp) and AWS (env-aws). Currently, this affects only the generated PDFs. To ensure compatability with the Jekyll version, the directives test for logical opposites. For example, if the attribute is NOT env-aws, display this content. For more information, see https://docs.asciidoctor.org/asciidoc/latest/directives/ifdef-ifndef/.
+
+CircleCI server v4.x is installed as a helm chart. The installation process is broken don into four phases. There is a validation step at the end of each phase, allowing you to confirm success before moving to the next phase. Depending on your requirements, phases 3 and 4 may include multiple steps. This installation guide assumes you have already read the  https://circleci.com/docs/2.0/server/overview/overview/[CircleCI Server v4.x Overview].
+
+.Installation Experience Flow Chart Phase 1
+image::server-install-flow-chart-phase1.png[Flow chart showing the installation flow for server 3.x with phase 1 highlighted]
+
+NOTE: In the following sections, replace any items or credentials displayed between `< >` with your details.
+
+[#install-required-software]
+== 1. Install required software
+Download and install the following software before continuing:
+
+[.table.table-striped]
+[cols=4*, options="header", stripes=even]
+|===
+| Tool
+| Version
+| Used for
+| Notes
+
+| https://www.terraform.io/downloads.html[Terraform]
+| {terraformversion} or greater
+| Infrastructure Management
+|
+
+| https://kubernetes.io/docs/tasks/tools/install-kubectl/[kubectl]
+| {kubectlversion} or greater
+| Kubernetes CLI
+|
+
+| https://helm.sh/[Helm]
+| {helmversion} or greater
+| Kubernetes Package Management
+|
+
+| https://github.com/databus23/helm-diff[Helm Diff]
+| {helmdiffversion} or greater
+| Helping with `values.yaml`` changes and updates
```

Comment:
It was confirmed by an engineer that we don't actually support .yaml files even though everyone was told we support both .yml and .yaml. Not sure if that matters here.

```suggestion
| Helping with `values.yaml` changes and updates
```